lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -316,11 +317,15 @@ export class StandardController implements IController {
 
         const initCorrelationId =
             request?.correlationId || this.getRequestCorrelationId();
-        const allowNativeBroker = this.config.system.allowNativeBroker;
-        const initMeasurement = this.performanceClient.startMeasurement(
-            PerformanceEvents.InitializeClientApplication,
-            initCorrelationId
-        );
+        let allowNativeBroker = false;
+        let initMeasurement;
+        if(this.isBrowserEnvironment) {
```

Comment:
I would suggest simplifying the changes here by doing an early return if `!this.isBrowserEnvironment`